apps/condo/domains/common/components/TrackingContext.tsx

```diff
@@ -106,7 +106,7 @@ const useTracking: IUseTracking = () => {
     }
 
     const getEventName = (eventType: TrackingEventType) => {
-        const [domainName, isDetail, suffix] = compact(route.split('/'))
+        const [domainName, isDetail, suffix, customPage] = compact(route.split('/'))
```

Comment:
`isDetail` looks like a boolean variable, but below we discover that the actual type is `string`. It discourages a bit :)

Reply:
Done